### PR TITLE
Some map.html and other improvements

### PR DESCRIPTION
**Pull request by dracos**
This loads areas as they come in rather than once at the end, which enables hover/popup info.
Also upgrade Leaflet, stop caching templates in development and improve rate limiter configuration.

### CODE REVIEW

File: project/settings.py [hunk 1]
```diff
@@ -200,13 +196,6 @@
             'mapit.context_processors.country',
             'mapit.context_processors.analytics',
         ),
-        # List of callables that know how to import templates from various sources.
-        'loaders': (
-            ('django.template.loaders.cached.Loader', (
-                'django.template.loaders.filesystem.Loader',
-                'django.template.loaders.app_directories.Loader',
-            )),
-        ),
```
Comment (mhl) [inline project/settings.py]: I'm sure I'm missing something simple, but how is it that this stops caching of templates in development only, and doesn't remve the cached loader entirefly?

Comment (dracos) [reply project/settings.py]: No, don't think you're missing anything, whoops!

Comment (dracos) [reply project/settings.py]: Ah, no, I wasn't! From Django 1.11, the cached loader is automatically enabled when DEBUG is False.

Comment (mhl) [reply project/settings.py]: Ah, OK - thanks :)

**Comment by dracos** [comment]
Yeah, we'll need to remove the existing script (added in 45f7ab4e) too. I've pushed a fixup to stop it highlighting on area page, thanks.

**Pull request merged**